.circleci/config.yml

```diff
@@ -329,6 +339,107 @@ jobs:
           success_message: ":rocket: RC version ${STREAMLIT_RELEASE_VERSION} was successful!"
           failure_message: ":blobonfire: RC version ${STREAMLIT_RELEASE_VERSION} failed to release"
 
+  build-deploy-release:
+    resource_class: large
+    docker:
+      - image: circleci/python:3.9.7
+    working_directory: ~/repo
+    steps:
+      - checkout:
+          name: Checkout Streamlit code
+
+      - run:
+          name: Install GitHub CLI tool and related dependencies
+          command: |
+            sudo apt-get install jq
+            /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+            eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+            brew install gh
+
+      - run:
+          name: Configure GitHub CLI Settings
+          command: echo 'export GH_REPO=$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME' >> $BASH_ENV
+
+      - run:
+          name: Look up the related GitHub PR number and branch name
+          command: |
+            eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
```

Comment:
I don't love this (you have to call the exec command whenever you want to access `gh` via brew), but couldn't get it to work by updating the path and putting it into the `$BASH_PROFILE` var in order to persist across sessions/commands 🤷🏻‍♀️